src/components/Stack/Stack.stories.tsx

```diff
@@ -0,0 +1,174 @@
+        direction: {
+            options: STACK_DIRECTIONS,
+            control: { type: 'select' },
+        },
+        spacing: {
+            options: SPACING_VALUES,
+            control: { type: 'select' },
+        },
+        justify: {
+            options: STACK_JUSTIFY_TYPES,
+            control: { type: 'select' },
+        },
+        alignItems: {
+            options: STACK_ALIGN_ITEMS_TYPES,
+            control: { type: 'select' },
+        },
+        children: { table: { disable: true } },
+    },
+    args: {
+        as: 'div',
+        bg: '',
+        color: '',
+        margin: 0,
+        padding: 12,
+        direction: 'column',
+        justify: 'start',
+        alignItems: 'start',
+        spacing: 8,
+        'data-test-id': 'custom-test-id',
+    },
+} as Meta<STACK_PROPS>;
+
+const Code = ({ children }: { children: ReactNode }): ReactElement => (
+    <code className="tw-bg-box-neutral tw-rounded tw-px-2 tw-text-box-neutral-inverse tw-text-s">{children}</code>
+);
+
+export const Stack: StoryFn<STACK_PROPS> = (args: STACK_PROPS) => {
+    const STACK_ITEM_CLASS_NAMES =
+        'tw-bg-box-positive-strong tw-text-box-positive-strong-inverse tw-rounded-md tw-px-3 tw-py-1';
+    return (
+        <StackComponent {...args}>
+            <div className={STACK_ITEM_CLASS_NAMES}>Test content.</div>
+            <div className={STACK_ITEM_CLASS_NAMES}>
+                First line
+                <br />
+                Next line
+                <br />
+                Last line
+            </div>
+            <div className={STACK_ITEM_CLASS_NAMES}>
+                All the good stuff is in the middle but the rest of it is necessary too.
+            </div>
+            <div className={STACK_ITEM_CLASS_NAMES}>Test!</div>
+        </StackComponent>
+    );
+};
+
+export const StackWithHorizontalDivider: StoryFn<STACK_PROPS> = (args: STACK_PROPS) => {
+    return <Stack {...args} />;
+};
+StackWithHorizontalDivider.args = {
+    divider: <Divider />,
+};
+
+export const StackWithVerticalDivider: StoryFn<STACK_PROPS> = (args: STACK_PROPS) => {
+    return <Stack {...args} />;
+};
+StackWithVerticalDivider.args = {
+    direction: 'row',
+    divider: <Divider vertical={true} />,
+};
+
+export const StackWithBoxAliasToken: StoryFn<STACK_PROPS> = (args: STACK_PROPS) => (
+    <StackComponent {...args}>
+        <p>
+            This is a full width container with bg and color{' '}
+            <a href="?path=/story/tokens--alias-tokens" className="tw-text-text-interactive" target="_blank">
+                Alias Tokens.
+            </a>
+        </p>
+        <p>
+            This story shows only Box Alias Tokens in controls, but you can pass any color token class to define bg and
+            color properties.
+        </p>
+    </StackComponent>
+);
+StackWithBoxAliasToken.args = {
+    bg: 'tw-bg-box-neutral',
+};
+
+export const StackAsSpanElement: StoryFn<STACK_PROPS> = (args: STACK_PROPS) => (
```

Comment:
That story doesn't seem to be very useful. As there is only one item within the Stack component, property like alignment or justify for example seems useless.

Reply:
@imoutaharik I think I forgot to push the changes. The span story is gone now.